pkg/gui/style/color.go

```diff
@@ -0,0 +1,32 @@
+package style
+
+import "github.com/gookit/color"
+
+type Color struct {
```

Comment:
I'm using this struct to handle the distinction between 16-bit and 256-bit colors. That way we don't need to duplicate code when it comes to handling non-color stuff like bold/underline/reverse